internal/lms/library/service_test.go

```diff
@@ -0,0 +1,126 @@
+package lms
+
+import (
+	"Demo/config"
+	"Demo/internal/entities/library"
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/gin-gonic/gin"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestStoreBook(t *testing.T) {
```

Comment:
where is failed test case?

Reply:
please create common functions for setup which will do all the initialization and delete all entities. use that at the start of each test. also use it to delete all entities.

Reply:
Done